src/lib/Microsoft.Health.Logger/Telemetry/TelemetryExtensions.cs

```diff
@@ -8,12 +8,16 @@
 using Microsoft.ApplicationInsights;
 using Microsoft.ApplicationInsights.DataContracts;
 using Microsoft.ApplicationInsights.Metrics;
+using Microsoft.Health.Common.Extension;
 
 namespace Microsoft.Health.Logging.Telemetry
 {
     public static class TelemetryExtensions
     {
         private static readonly string _namespace = MetricIdentifier.DefaultMetricNamespace;
+        private const string _helplinkAttribute = "helpLink";
+        private const string _messageAttribute = "message";
+        private const string _LogForwardingAttribute = "logToCustomerIot";
```

Comment:
nit: casing inconsistency.  We should have style cop rules that will enforce this but I believe private constants the style should be camel case.

Reply:
Thanks for catching this!